sbc_translation/data/mail_template.xml

```diff
@@ -36,5 +36,11 @@
         </table>
         <hr/>
         <p><a t-att-href="object.translation_url">Open the letter in the translation platform</a></p>
+        <hr/>
+        <br/><p>Thank you for your help,
+        % if user and user.signature:
+        ${user.signature | safe}
+        % endif
```

Comment:
```suggestion
        
```
@ecino I tried to add the user signature but when testing it doesn't work. Can you correct it for me ? Thanks in advance